schemas/storage.sql

```diff
@@ -125,7 +125,7 @@ CREATE TABLE UserIdentities (
     INDEX(NAME)
 );
 
-DROP FUNCTION IF EXISTS UserIdentityLookup;
+DROP FUNCTION IF EXISTS UserIdentitiesLookup;
```

Comment:
Why change this? Is this intended to change the name of the function `UserIdentityLookup` to `UserIdentitiesLookup`?

If so
- Shouldn't the subsequent `CREATE FUNCTION UserIdentityLookup` should also be changed.
- Presumably we'd need an update schema script similar to https://github.com/apel/apel/blob/dev/scripts/update-1.7.0-1.8.2.sql